### ISSUE TITLE
no ability to add version to doc

### ISSUE THREAD

**Issue by acordiner92** [airtasker/spot#652]
### Current behavior:
Currently there is no ability to be able to add a version to the doc (aka setting the version in the info section of open api spec).
### Desired behavior:
To have the option to be able to set the version.
A potential solution maybe in the @api component have a info object which has a version field that can be set e.g
```
@api({
  name: "My API",
  info: {
     version: 0.2.0
  }
})
```
### Steps to reproduce:
N/A

**Comment by committedteadrinker** [comment]
Hi @acordiner92,

The concept of `info` is more aligned to what OpenAPI has. Would suggest to support this syntax instead:
```
@api({
  name: "My API",
  version: 0.2.0
})
```

Please create a PR for this as well for further discussion on the implementation side. Thanks!

**Comment by committedteadrinker** [comment]
@theRingleman @acordiner92 I'm closing this as it has been addressed on the PR and released on version 1.1.3.
